person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  sttr_reader :id
+  sttr_accessor :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ OPTIONAL] I recommend here to add parenthesis here when you are calling a method so kindly add them to be somthing like this 💯✨🔥✅

`def can_use_services? 
    is_of_age?() || @parent_permission
  end
`